content/install_maintain/installation/installing-cluster.md

```diff
@@ -76,6 +76,43 @@ cfy cluster update-profile
  - Make sure the PostgreSQL instance is publicly available and reachable from the local Cloudify Manager machine.
  - Download the CA certificate installed on the PostgreSQL DB instance into the local Cloudify Manager machine.
  
+##### Azure DBaaS for Postgres
+
+Azure has support for DBaaS for Postgres.  
+The DBaaS of Azure supports a clustered instance and a single instance available for resizing on demand.  
+As opposed to other DBaaS vendors, Azure doesn't give access to the `postgres` user with SuperUser privileges, so while working with Azure DBaaS is fully supported, the configuration is a bit different than regular Postgres installations.  
+
+In case you have decided to use [Azure DBaaS for Postgres](https://docs.microsoft.com/en-us/azure/postgresql/), either a single instance or a clustered instance, you will need to update Cloudify manager accordingly.  
```

Comment:
Using Azure DBaaS (either the single instance or the clustered instance), requires specific setup changes to the Cloudify manager configuration.